test/dlc_tests/sanity/test_anaconda.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from invoke.context import Context
+from test import test_utils
+
+
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("N/A")
+@pytest.mark.integration("anaconda_removal")
+def test_repo_anaconda_not_present(image):
+    """ Test to see if all packages installed in the image do not come from repo.anaconda.com """
+    ctx = Context()
+    container_name = test_utils.get_container_name("anaconda", image)
+    test_utils.start_container(container_name, image, ctx)
```

Comment:
nit: I believe there is a stop_and_remove_container function in test utils as well - you can add this to the end of the function or as a [pytest finalizer](https://docs.pytest.org/en/6.2.x/fixture.html#adding-finalizers-directly). This will just ensure that the containers don't persist on the codebuild host and potentially contribute to memory issues when running sanity tests

Reply:
update to use a try finally statement